docs/env-vars.md

```diff
@@ -9,6 +9,11 @@ Typeagent currently supports two families of environment variables:
 - Those for (public) OpenAI servers.
 - Those for the Azure OpenAI service.
 
+**Note:** The MCP server (`typeagent/mcp/server.py`) uses MCP's sampling API
```

Comment:
Let's not change this file. When we document the MCP server we'll put the note there.